algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -66,10 +66,13 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
                                 sourceGene,
                                 sourceEvent)) {
 
+                            Set<String> sourceUrls = Sets.newHashSet();
+                            sourceUrls.add("https://ckbhome.jax.org/profileResponse/advancedEvidenceFind?molecularProfileId=" + entry.profileId());
```

Comment:
You could use `Collections.singleton("<url>")` instead

Reply:
solved